frontend/src/style/App.css

```diff
@@ -77,4 +78,19 @@
 
 .table-header {
   background: rgb(75,160,181);
-}
\ No newline at end of file
+}
+/*Nav styles*/
+
+.h-hub-brand {
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.h-hub-nav-link {
+  text-decoration: none;
+  color: white;
+}
+
+.h-hub-nav-link:hover {
+  text-decoration: none;
+  color: rgb(75,160,181);
+}
```

Comment:
Could you use bootstrap for some of this instead of css?  For example, you might consider using the class 'text-white' and the bootstrap colors.  

https://getbootstrap.com/docs/4.0/utilities/colors/